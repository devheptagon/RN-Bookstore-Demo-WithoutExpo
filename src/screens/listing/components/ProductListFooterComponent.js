import React from "react";
import PropTypes from "prop-types";
import { PacmanIndicator } from "react-native-indicators";
import FlexView from "../../components/FlexView";
import Theme from "../../../constants/Theme";

export const ProductListFooterComponent = props => {
  return (
    <FlexView centerAll>
      {props.isVisible ? (
        <PacmanIndicator color={Theme.Loading.PacmanColor} />
      ) : null}
    </FlexView>
  );
};

ProductListFooterComponent.propTypes = {
  isVisible: PropTypes.bool
};

export default ProductListFooterComponent;
