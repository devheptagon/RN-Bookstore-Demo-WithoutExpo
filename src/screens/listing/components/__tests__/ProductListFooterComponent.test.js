import React from 'react';
import {ProductListFooterComponent} from '../ProductListFooterComponent';
import renderer from 'react-test-renderer';

test('ProductListFooterComponent renders correctly', () => {
  const tree = renderer.create(<ProductListFooterComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});