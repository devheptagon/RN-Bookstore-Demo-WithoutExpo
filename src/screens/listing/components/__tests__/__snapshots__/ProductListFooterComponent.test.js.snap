// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductListFooterComponent renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
/>
`;
